app/controllers/sessions_controller.rb

```diff
@@ -41,7 +43,7 @@ def lookup_login
                   set_login_info(username_or_email: @handler_result.outputs.username_or_email,
                                  names: @handler_result.outputs.names,
                                  providers: @handler_result.outputs.providers)
-                  redirect_to :authenticate
+                  render :authenticate
```

Comment:
Yeah, I swapped it to `render` so it'd retain the email value for the next form's hidden field, where it'd eventually be passed onto the `custom_identity` strategy.   However we could also use the signed cookie for that as well.  I've made a change for that.